### PR TITLE
Restructure the project to not use libraries internally

### PR DESCRIPTION
**Pull request by huntears**
Cross dependencies were impossible, so I got rid of the libraries.

### CODE REVIEW

File: cubic-server/logging/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,49 @@
+target_sources (${CMAKE_PROJECT_NAME} PRIVATE
+        TimeFormatter.cpp
```
Comment (Trompettesib) [inline cubic-server/logging/CMakeLists.txt:2]: maybe there is a missing hpp here

File: cubic-server/math/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+target_sources (${CMAKE_PROJECT_NAME} PRIVATE
+        Math.cpp
```
Comment (Trompettesib) [inline cubic-server/math/CMakeLists.txt:2]: maybe there is a missing hpp here

File: cubic-server/protocol/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Create a library called "Hello" which includes the source file "hello.cxx".
+# The extension is already found. Any number of sources could be listed here.
+target_sources (${CMAKE_PROJECT_NAME} PRIVATE
```
Comment (Trompettesib) [inline cubic-server/protocol/CMakeLists.txt:3]: maybe there is missing hpp

**Review by Trompettesib — COMMENTED** [review]
LGTM even if I loved our last archi with libraries

**Review by Trompettesib — APPROVED** [review]
LGTM

**Review by STMiki — APPROVED** [review]

**Pull request merged**